sirepo/template/template_common.py

```diff
@@ -207,7 +207,10 @@ def validate_model(model_data, model_schema, enum_info):
             raise Exception('no value for field "{}" and no default value in schema'.format(k))
         if field_type in enum_info:
             if str(value) not in enum_info[field_type]:
-                raise Exception('invalid enum value: {} for {}'.format(value, k))
+                # Check a comma-delimited string against the enumeration
+                for item in re.split(",\s*", str(value)):
+                    if item not in enum_info[field_type]:
+                        raise Exception('invalid enum value: {} for {}'.format(value, k))
```

Comment:
Exception() is not a good thing. At a minimum, use `assert` or raise AssertionError. Exception is generic so it can't be caught by type. Assertion indicates that you are throwing a program error, not a generic Exception (e.g. IOException subclasses Exception). 

As to the format statement, it should be of the form: "<value>:  <error>", e.g. 

```python
assert item in enum_info[field_type], \
    '{}: invalid enum "{}" value for field "{}"'.format(item, field_type, k)
```